### DISCUSSION TITLE
Proposal Process for New Games in the Org

### DISCUSSION THREAD

**Discussion by Yonodactyl** [Godot-Community-Games-Org/.github#2] [Ideas]
How do we feel about a proposal process for new games to gauge interest and establish a repository? Here is what I'm thinking:

### How Proposals Will Work:
1. **Create a Discussion Post:**  
   If you have a game idea, start a new discussion in the org (or specific repo). Use a title like “Proposal: [Game Name]” to keep things organized.

2. **Follow the Pitch Format:**  
   Keep it simple but informative! Your post should include:
   - **Title:** Name of your game or a placeholder.
   - **Concept:** A brief summary of what the game is about (genre, core mechanics, themes).
   - **Unique Mechanics/Features:** What makes this game stand out?
   - **Tech Stack:** Any special tools/plugins needed?  
   - **Scope:** Rough timeline or how big the project will be (e.g., quick jam vs bigger effort).
   - **Team Requirements:** Any specific roles you need help with (art, sound, etc.)?

   Example:

   **Title:** Duck Quest 2: The Feather Awakens  
   **Concept:** Sequel to Duck Quest. The player solves puzzles by switching between light and dark realms.  
   **Unique Mechanics/Features:** Adding a new water-dive mechanic for hidden feathers.  
   **Tech Stack:** Godot + Steam integration.  
   **Scope:** Medium project (~2-3 months).  
   **Team Requirements:** Need 1-2 artists and sound designers!

3. **Collaborative Feedback:**  
   Let’s leave each proposal open for feedback for about a week. This is the time to brainstorm, suggest changes, or throw out new ideas.

4. **Voting:**  
   After the discussion period, we’ll use reactions (👍👎) to gauge interest and vote. If there’s enough interest, we’ll move forward.

5. **Getting Started:**  
   Once a game proposal gets traction, we’ll:
   - Create a new repo for the game.
   - Assign team roles to those interested.
   - Set up a roadmap (optional but helpful for long-term projects).

***

I'm open to other ideas on making this a good starting point!

**Comment by GodotCommunityGames** [comment]
Nice Yono, that's a great idea.

**Comment by TheBardWhoCodes** [comment]
I wonder if it's worth it to decide on a language (C#, GDScript, any) as part of this or decide on that later.

**Reply by Yonodactyl** [reply]
I think language can deff be part of this process. We can in theory use both in one project, but some might not be familiar with the other so it's good to set the project specification so that new people wanting to join the project can choose to not work on it if it's out of their element!

**Reply by GodotCommunityGames** [reply]
Yeah should be on a project by project basis, but set out from the start if the game is gonna be GDScript or C# or any. Also possible that a GDScritpt-only game could have a little bit of C# code for performance sake if say there is some really computationally intensive code that a C# dev wants to run faster than if it was written in GDScript.

But yeah setting out which langauge(s) will be used at the start of each project will be good so that people know what to expect if they want to work on a project.

**Comment by ghost** [comment]
Very nice idea ! Should be a more helpful way of getting things together. I like it
